requirements-dev.txt

```diff
@@ -8,3 +8,6 @@ splinter>=0.13.0
 -e git+https://github.com/qld-gov-au/ckanext-data-qld-theme@develop#egg=ckanext-data-qld-theme
 -e git+https://github.com/qld-gov-au/ckanext-odi-certificates@develop#egg=ckanext-odi-certificates
 -e git+https://github.com/ckan/ckanext-dcat.git#egg=ckanext-dcat
+-e git+https://github.com/qld-gov-au/ckanext-archiver#egg=ckanext-archiver
```

Comment:
These should use the `develop` branch.

Reply:
@ThrawnCA I have updated ckanext-archiver & ckanext-report to use the 'develop' branch.
Once the https://github.com/qld-gov-au/ckanext-qa/pull/1 has been approved and merged I will update the QA extension to 'https://github.com/qld-gov-au/ckanext-qa@develop#egg=ckanext-qa'